app/src/main/java/com/example/poste/activities/DashboardActivity.java

```diff
@@ -207,23 +210,14 @@ private void configureWindow() {
         // Setup refresh
         if (swipeRefreshLayout == null) {
             swipeRefreshLayout = (SwipeRefreshLayout) findViewById(R.id.swipe_refresh_layout);
-            swipeRefreshLayout.setOnRefreshListener(this::reloadDashboard);
+            //swipeRefreshLayout.setOnRefreshListener(this::reloadDashboard);
+            swipeRefreshLayout.setOnRefreshListener(() -> {
+                updateData();
+                swipeRefreshLayout.setRefreshing(false);
+            });
```

Comment:
Swipe refresh will now just update data; the recyclerview will update automatically without reloading activity.